asgi_rabbitmq/core.py

```diff
@@ -939,15 +906,14 @@ def __init__(self,
         # Connection thread will be started on first method access.
         self._thread = self.Thread(url, expiry, group_expiry,
                                    self.get_capacity, crypter)
+        self._thread.start()
```

Comment:
Pika connection tries to connect to RabbitMQ right after a channel layer initialization, and it's split into 2 parts:
- instantiating a connection, 
- running IOLoop to handle server responses

There's 10 seconds timeout to complete the connection setup. When the timeout is running out, server responds with handshake timeout and breaks the connection, so all next attempts to use it obviously fail. If any messages are sent within the 10 seconds - it triggers running IOLoop and everything works fine.

2nd part of the setup is delayed purposely until the first message comes and triggers IOLoop run, see
(in a word, it was done to allow a custom setup to instantiate layers w/o connecting to RabbitMQ at once):
- https://github.com/proofit404/asgi_rabbitmq/commit/4c387ef2fc4447aac2f9b17b32b6bee275a0d97c
- https://github.com/proofit404/asgi_rabbitmq/issues/5#issuecomment-301802605

My guess it wasn't a problem in the past as we're using Twisted extension in the layer, which scheduled the first message (`RECEIVE`) instantly from the beginning. However, while the Twisted part is not used anymore, we have to keep the similar model and run IOLoop closer to the connection instantiation.

I see 2 options here: run the connection thread right after initialization, a) when creating a channel layer or b) when the first message comes. Setting connection takes time, so the first option looks a bit better, but I wonder if you think in a different way.